apps/storefront/src/pages/OrderDetail/components/OrderBilling.tsx

```diff
@@ -21,6 +27,47 @@ export default function OrderBilling({ isCurrentCompany }: OrderBillingProps) {
 
   const b3Lang = useB3Lang();
 
+  const [isDigitalDownloadOpen, setIsDigitalDownloadOpen] = useState(false);
+  const [digitalProducts, setDigitalProducts] = useState<OrderProductItem[]>([]);
+  const [currentDigitalProduct, setCurrentDigitalProduct] = useState<ProductItem | undefined>(
+    undefined,
+  );
+
+  useEffect(() => {
+    const getDigitalProductsInformation = async (id: number | string) => {
+      const elements = await getDigitalDownloadElements(id);
+
+      if (!elements.length) {
+        return;
+      }
+
+      const digitalProductsData = elements.map((item) => item.node);
+
+      const digitalProducts = billings[0].digitalProducts.map((product: OrderProductItem) => {
```

Comment:
Per our mobbing discussion:

- we should just introduce a new state variable called `billingAddress` and `digitalProducts` (`billings` being an array of 1 item that has these two values is unnecessary)
- combine usage of `convertB2BOrderDetails` and `convertBCOrderDetails` as they are the same function 😕